eng/versioning/external_dependencies.txt

```diff
@@ -8,6 +8,7 @@ com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.10.1
 com.fasterxml.uuid:java-uuid-generator;3.2.0
 com.github.spotbugs:spotbugs;4.0.0-beta3
 com.github.spotbugs:spotbugs-maven-plugin;3.1.12.2
+com.google.code.findbugs:annotations;2.0.1
```

Comment:
We shouldn't be needing findbugs dependencies - we use spotbugs in track two and it would ideally be used here too.